demo/mkldnn_quant/README_en.md

```diff
@@ -0,0 +1,181 @@
+```
+python save_quant_model.py --quant_model_path=/PATH/TO/SAVE/FLOAT32/quant/MODEL --int8_model_save_path=/PATH/TO/SAVE/INT8/MODEL
+```
+
+**Available options in the above command and their descriptions are as follows:**
+- **quant_model_path:** input model path, required. A quant model for quantifying training output.
+- **int8_model_save_path:** The final INT8 model output path after the quant model is optimized and quantized by DNNL.
+- **ops_to_quantize:** A comma separated list of specified op types to be quantified. Optional, default is empty, empty means to quantify all quantiable ops. At present, for the image classification and natural language processing models listed in Benchmark, quantizing all quantiable ops can obtain the best accuracy and performance, so it is recommended that users not set this parameter.
+- **--op_ids_to_skip:** A list of op id numbers separated by commas, optional. Default value is none. The op numbers in this list will not be quantified and adopt the FP32 type. To get the ID of a specific op, first run the script using the `--debug` option, and open the generated file `qat_int8_cpu_quantize_placement_pass.dot` to find the op that does not need to be quantified, and the ID number is in parentheses after the Op name.
+- **--debug:** Generate models graph or not. Add this option to generate a series of *.dot files containing model drawings after each conversion step. For the description of DOT format, please refer to [DOT](https://graphviz.gitlab.io/_pages/doc/info/lang.html). To open the `*.dot` file, please use any Graphviz tool available on the system(such as the `xdot` tool on Linux or the `dot` tool on Windows. For Graphviz documentation, see [Graphviz](http://www. graphviz.org/documentation/).
+  
+- **Note:**
+  - The DNNL supported quantizable ops are `conv2d`, `depthwise_conv2d`, `fc`, `matmul`, `pool2d`, `reshape2`, `transpose2`, `scale`, `concat`.
+  - If you set `--op_ids_to_skip`, you only need to pass in all the quantized op ID number you want to keep FP32 type.
+  - Sometimes quantizing all ops does not necessarily result in optimal performance. For example: If an op is a single INT8 op, before and after the op are float32 op, then in order to quantify this op, you need to do quantize first, then run INT8 op, and then dequantize, which may lead to the final performance is not as good as keeping the op fp32 op. If the user has poor performance using the default settings, you can observe whether this model has a separate INT8 op, select different combinations of `ops_to_quantize`, or you can exclude some quantiable op IDs through `--op_ids_to_skip`, and run several times to get the most Good settings.
```

Comment:
Maybe:
"Quantization yields best performance and accuracy when a long sequences of consecutive quantized operators are present in the model. When a model contains quantizable operators surrounded by non-quantizable operators, quantization of the single operators (or very short sequences of them) can give no speedup or even cause drop in performance because of frequent quantizing and dequantizing the data. In that case user can tweak the quantization process by limiting it to the particular types of operators (using the `--ops_to_quantize` option) or by disabling quantization of particular operators, e.g. the single ones or in short sequences (using the `--op_ids_to_skip` option)."